projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_button-group.scss

```diff
@@ -48,18 +57,21 @@ $_light-button-group: (
         igx-color: ('grays', 400)
     ),
 
+    item-selected-hover-background: (
+        igx-color: ('grays', 500)
+    ),
+
     item-selected-border-color: (
-        igx-color: ('grays', 600),
-        rgba: .12
+        igx-color: ('grays', 300)
     ),
 
     disabled-text-color: (
         igx-color: ('grays', 400)
     ),
 
     disabled-background-color: (
-        igx-color: ('grays', 100)
+        igx-color: ('grays', 50)
     ),
 
-    elevation: 2
+    elevation: 0
```

Comment:
Doesn't this introduce too big of a visual breaking change? I'd keep as it was before.